tests/testthat/test-srp_model.R

```diff
@@ -37,19 +37,19 @@ test_that("private function is_valid throws correct errors", {
   expect_error(
```

Comment:
we need a test case showing that a single-group design can be instantiated. that was the problem in the first place, right?

Reply:
I tried single-group, like we did. It works